app/views/rockles/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<ul>
+
+  <% @rockles.each do |rockle| %>
+
+<li><%= rockle.name %>:
```

Comment:
@gdomingu use ` to surround your code snippets! `<code>`

I think we can leave it as a list for now until we know better what we want it to look like.